### ISSUE TITLE
Missing `linux/ppc64le` for Golang 1.18 image

### ISSUE THREAD

**Issue by jauderho** [docker-library/golang#416]
Go 1.18 was just released but it appears that the build job for linux/ppc64le have failed and need to be respun.

This is in reference to https://github.com/docker-library/official-images/pull/12045#issuecomment-1068723763 

Per @DasSkelett,

* https://doi-janky.infosiftr.net/job/multiarch/view/images/view/golang/job/ppc64le/job/golang/
* https://doi-janky.infosiftr.net/job/multiarch/view/images/view/golang/job/ppc64le/job/golang/587/console

FWIW, it looks like PR docker-library/official-images#12055 is inbound so it's unclear if we want to wait until that rolls out.

**Comment by tianon** [comment]
The failure looks like a pretty consistent failure in either `go` from Alpine itself (installed fresh within that layer, so https://github.com/docker-library/official-images/pull/12055 is not likely to cause any change), or in the bootstrap version of `go` that gets built from Alpine's `go`:

```console
...
+ ./make.bash
Building Go cmd/dist using /usr/lib/go. (go1.16.10 linux/ppc64le)
Building Go toolchain1 using /usr/lib/go.
Building Go bootstrap cmd/go (go_bootstrap) using Go toolchain1.
Building Go toolchain2 using go_bootstrap and Go toolchain1.
Building Go toolchain3 using go_bootstrap and Go toolchain2.
Building packages and commands for linux/ppc64le.
go tool dist: FAILED: /usr/local/go/bin/go list -gcflags=all= -ldflags=all= -f={{if .Stale}}	STALE {{.ImportPath}}: {{.StaleReason}}{{end}} std cmd: signal: segmentation fault (core dumped)
```

**Comment by DasSkelett** [comment]
There's now an upstream Go issue tracking this, the Alpine people have hit the same trying to build the package for their repository:
https://github.com/golang/go/issues/51787
On EOF their build logs: https://gitlab.alpinelinux.org/alpine/aports/-/jobs/661598#L1073

**Comment by jauderho** [comment]
Looks like ppc64le builds are showing up now BUT only for certain tags. ppc64le builds are available for `alpine` and `alpine3.15` but not available for `1.18.0-alpine` or `1.18.0-alpine3.15`

See https://hub.docker.com/_/golang?tab=tags&page=1&name=alpine

**Comment by yosifkit** [comment]
> Looks like ppc64le builds are showing up now BUT only for certain tags. ppc64le builds are available for `alpine` and `alpine3.15` but not available for `1.18.0-alpine` or `1.18.0-alpine3.15`

The manifest lists that are the combination of multiple architectures into a single tag are built from the tags are available in each architecture (because some architectures take longer to build than others). For example, here is the Docker Hub [ppc64le/golang](https://hub.docker.com/r/ppc64le/golang) repo. `ppc64le` images are pushed there and then another job combines them into the main manifest under https://hub.docker.com/_/golang. The result is that rather than architectures disappearing and reappearing on the more general tags, the slow (or failing-to-build architectures) instead happen to point to the previous image. In this case, it is the [`golang:1.17.8-alpine` image](https://hub.docker.com/layers/golang/library/golang/alpine3.15/images/sha256-830d31adef4290dcdacff119e2e988a47bddd5b5e437364dd247edc4d0d8741a?context=explore).
